comm.py

```diff
@@ -89,12 +89,32 @@ def send_to_simulator(msg):
     except Exception as e:
         print('Unexpected error occured:', e)
 
+def convert_command_syntax(cmd: str):
```

Comment:
```suggestion
def convert_command_syntax(cmd):
```

For the sake of consistency, lets keep type definitions out for now. Once we have some extra time or need additional tasks we can start getting into refactors such as adding type definitions to python modules, converting react class components to functional components, etc.